code/modules/weather/weather_fsm_states.dm

```diff
@@ -199,12 +207,37 @@
 		"The wind howls around you.",
 		"Swirling sand obscures your vision."
 	)
-	protected_messages = list("Stinging sand blows against $ITEM$.")
+	protected_messages = list("$ITEM$ shields you from the howling sandstorm.")
 	ambient_sounds = list('sound/effects/weather/sandstorm.ogg')
 
 /singleton/state/weather/sandstorm/handle_exposure_effects(mob/living/M, obj/abstract/weather_system/weather)
-	to_chat(M, SPAN_DANGER("You are blasted by a gust of stinging sand!"))
-	M.adjustBruteLoss(rand(1,5))
+	if((istype(M, /mob/living/simple_animal) && M.faction == "Moghes") || M.get_species(FALSE) == SPECIES_MONKEY_UNATHI) //Not realistic that they're just immune, but we don't have AI for simplemobs seeking shelter from storms.
```

Comment:
checking for a faction rather than a variable/flag on the animal seems a bit weird...?

Reply:
This was done purely out of laziness, has been changed